Makefile.am

```diff
@@ -110,8 +110,11 @@ BATS_LOG_DRIVER = $(tap_driver)
 
 TESTS = $(dist_check_SCRIPTS)
 
-BATS = \
+BATS = test/functional/bundleadd/add-alias-basic.bats
+
+BBATS = \
```

Comment:
I don't follow why you are doing this. Why are you adding this test twice? and why are you renaming the BATS tests as BBATS? is this something temporary so only your tests run while you figure out why they are failing?

Reply:
Temporary for the test failures yes. Commit has been removed.